Platform support updates for 23.02.

bdice opened the pull request:
This PR updates our release selector for 23.02. @charlesbluca noticed that our Python versions have been incorrect for nightly versions of 23.02 (Python 3.9 has been dropped but has been showing on the release selector for nightly users).

@sevagh Can you please review the pip changes?

I would appreciate a critical eye from other reviewers as well. 🙏

raydouglass commented on start.md:
```diff
@@ -119,7 +119,7 @@ All provisioned systems need to be RAPIDS capable. Here's what is required:
 - <i class="fas fa-check-circle text-purple"></i> [CUDA 11.2](https://developer.nvidia.com/cuda-11.2.0-download-archive){: target="_blank"} with Driver 460.27.03 or newer
 - <i class="fas fa-check-circle text-purple"></i> [CUDA 11.4](https://developer.nvidia.com/cuda-11-4-0-download-archive){: target="_blank"} with Driver 470.42.01 or newer
-- <i class="fas fa-check-circle text-purple"></i> [CUDA 11.5](https://developer.nvidia.com/cuda-11-5-0-download-archive){: target="_blank"} with Driver 495.29.05 or newer
+- <i class="fas fa-check-circle text-purple"></i> [CUDA 11.8](https://developer.nvidia.com/cuda-11-8-0-download-archive){: target="_blank"} with Driver 520.61.05 or newer
```
We are still supporting CUDA 11.5, so I think you leave 11.5 and add 11.8 here.

bdice replied:
Good call. I always forget this because we dropped 11.5 from our PR CI. I think it’s still tested in nightlies?

bdice replied:
Fixed in d7fb44346379c3542d2df53566c62e631a11ebf7.

raydouglass commented on _includes/selector.html:
```diff
@@ -138,9 +138,9 @@
-            cuda_vers: ["11.2", "11.4", "11.5"],
-            os_vers: ["Ubuntu 18.04", "Ubuntu 20.04", "CentOS 7", "Rocky Linux 8"],
+            python_vers: ["3.8", "3.10"],
+            cuda_vers: ["11.2", "11.4", "11.8"],
```
Should leave 11.5 in and add 11.8

raydouglass reviewed (changes requested):
Requesting changes to block merging until the release is complete

vyasr reviewed (approved):
FWIW everything looks right to me. Only question is why in the WSL section the recommendations include Ubuntu 20.04 and 22.04, whereas before 18.04 wasn't included. Were there actual limitations with 18.04, was it not available via WSL2, or should we be continuing to recommend only one (the newest) Ubuntu version?

bdice commented:
> FWIW everything looks right to me. Only question is why in the WSL section the recommendations include Ubuntu 20.04 and 22.04, whereas before 18.04 wasn't included. Were there actual limitations with 18.04, was it not available via WSL2, or should we be continuing to recommend only one (the newest) Ubuntu version?

I think we only tested on 20.04 in the pre-GA phase, and now we've dropped 18.04 so it's irrelevant. But now that we've hit "GA" quality releases and added 22.04 support, I didn't know if I should _replace_ 20.04 with 22.04 (which I run regularly and know to work) or just list both. I am fine with either approach and would defer to the opinions of @aravenel @raydouglass @taureandyernv @jarmak-nv. Should we only list the latest Ubuntu version in our WSL instructions?

bdice commented on _includes/selector.html:
```diff
@@ -288,7 +288,7 @@
                 var cmd = this.highlightCmd("docker") + " pull " + fullImg + "\n" +
                     this.highlightCmd("docker") + " run --" + this.highlightFlag("gpus") + " all --" + this.highlightFlag("rm") + " -" + this.highlightFlag("it") + " \\\n" +
-                    indent + "--" + this.highlightFlag("shm-size") + "=1g --" + this.highlightFlag("ulimit") + " memlock=-1" + " \\\n" +
+                    indent + "--" + this.highlightFlag("shm-size") + "=1g --" + this.highlightFlag("ulimit") + " memlock=-1 --" + this.highlightFlag("ulimit") + " stack=67108864" + " \\\n" +
```
@taureandyernv Where does this magic value of $2^{26}$ come from? Is it just "a large number" or does it mean something?

jarmak-nv commented:
@bdice re: WSL Ubuntu Version

I think we should be good to just say use the latest ubuntu. It was _very slightly_ awkward before with Ubuntu 22.04 including Python 3.10 but now that we're on 3.10 it's aligned well. Constraining our recommendation reduces expected support as well which is not a bad thing for us.

bdice commented:
> @bdice re: WSL Ubuntu Version
> 
> I think we should be good to just say use the latest ubuntu. It was _very slightly_ awkward before with Ubuntu 22.04 including Python 3.10 but now that we're on 3.10 it's aligned well. Constraining our recommendation reduces expected support as well which is not a bad thing for us.

Agreed. Done in 46f77656471292d083a6bc93b9ae437bb4bdc828.

beckernick reviewed (approved):
This LGTM pending @bdice 's question about the magic number for ulimit and @jarmak-nv 's PR https://github.com/rapidsai/rapids.ai/pull/295
